iamlistening/handler/client.py

```diff
@@ -100,7 +100,7 @@ async def handle_message(self, message_content):
 
         if self.is_connected:
             self.latest_message = message_content
-            logger.debug("Frasier👂 on {}: {}", self.platform, message_content)
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider conditional logging instead of commenting out.

Commenting out debug logs can be useful for reducing log noise, but it might be more beneficial to implement a conditional logging mechanism based on the environment or a configuration setting. This way, you can easily toggle the verbosity of logs without altering the code.

```suggestion
import os

if os.getenv('LOG_LEVEL', 'INFO') == 'DEBUG':
    logger.debug("Frasier👂 on {}: {}", self.platform, message_content)
```